plugins/gatsby-source-cd-airtable/src/airtable-connection.ts

```diff
@@ -0,0 +1,29 @@
+import fetch, { RequestInit } from 'node-fetch'
+import { AirTableProject } from './interfaces/airtable-project'
+
+export class AirTableConnection {
```

Comment:
Je nějaký důvod, proč mít tohle připojení jako třídu? Plánujeme například mít více instancí, které by se nějak lišily a něco se na nich volalo opakovaně? Zatím tam žádné výhody nevidím, `loadData` by mohla být jenom funkce.

Reply:
Udělal jsem to jako třídu, aby to bylo do budoucna lépe rozšiřitelné (a daly se např. načítat i další data, ne jenom projekty). Můžu to klidně ale upravit, aby to byla jenom funkce.

Reply:
Jestli můžeš, tak to prosím předělej, ať zbytečně nemícháme paradigmata :-)